cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,55 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SetttingsObject from '../support/pages/settings.pageObject';
 
+const settingPageObject = new SetttingsObject();
```

Comment:
The instantiation of `SetttingsObject` should be corrected to `SettingsObject` to match the corrected class name.